client/data/agency-dashboard/use-fetch-dashboard-sites.ts

```diff
@@ -45,6 +59,7 @@ const useFetchDashboardSites = (
 					...( currentPage && { page: currentPage } ),
 					...agencyDashboardFilterToQueryObject( filter ),
 					...agencyDashboardSortToQueryObject( sort ),
+					per_page: per_page ?? 20,
```

Comment:
`...( per_page && { per_page } ),` ?

Reply:
This also works, so can be included in an update, thanks.